qrhl/qrhl.el

```diff
@@ -0,0 +1,115 @@
+;; Author: Dominique Unruh
+
+(require 'proof)
+
+(load-library "qrhl-input")
```

Comment:
You should probably use require instead of load-library.
Moreover this kind of library should be opt-in, as it changes the input method, but some users may have other ways to input symbols. You should maybe have an option to disable it.

Reply:
I'm using require now (847492f7). I also made the input method configurable (0eb165d) but the `require` is still unconditional. That's because I did not find a way to autoload `qrhl-input.el` if the input method `qrhl` is chosen. (Note: if the input method is not chosen, then this `require` has only the effect of creating the input method, not activating it.)